params.py

```diff
@@ -66,6 +66,13 @@
                                            partial(enfsi_data, year=2017),
                                                                           ]))],
             'fraction_test': .2,
+        },
+        'lfw': {
+            'datasets': [DataFunctions(image_provider=None,
+                                       pair_provider=partial(combine_paired_data, pair_providers=[
```

Comment:
pair_provider=lfw_data 
would work, no need for the combine_paired_data or partial (they are needed to glue together the several enfsi years)